crates/re_viewer_context/src/lib.rs

```diff
@@ -2,26 +2,42 @@
 //!
 //! This crate contains data structures that are shared with most modules of the viewer.
 
+mod annotations;
 mod app_options;
 mod caches;
 mod component_ui_registry;
 mod item;
+mod scene_query;
 mod selection_history;
 mod selection_state;
+mod tensor;
 mod time_control;
+mod utils;
 mod viewer_context;
 
+// TODO(andreas): Move to its own crate?
+pub mod gpu_bridge;
```

Comment:
the thought has occurred to me too - a re_renderer <-> re_log_types bridge crate.